### PR TITLE
[KGP-2123] Tooltip for Not Reached

### PR DESCRIPTION
**Pull request by nhd98z**

### CODE REVIEW

File: src/pages/KyberDAO/Vote/ProposalItem/VoteInformation.tsx [hunk 1]
```diff
@@ -1,17 +1,23 @@
...
-import { RowBetween } from 'components/Row'
+import Row, { RowBetween } from 'components/Row'
 import { useActiveWeb3React } from 'hooks'
 import { useVotingInfo } from 'hooks/kyberdao'
 import { ProposalDetail } from 'hooks/kyberdao/types'
 import useTheme from 'hooks/useTheme'
 
+import WarningIcon from '../../../../components/Icons/WarningIcon'
+import { MouseoverTooltip } from '../../../../components/Tooltip'
+import { BIPS_BASE } from '../../../../constants'
```
Comment (namgold) [inline src/pages/KyberDAO/Vote/ProposalItem/VoteInformation.tsx:19]: :danh:
```suggestion
import WarningIcon from 'components/Icons/WarningIcon'
import { MouseoverTooltip } from 'components/Tooltip'
import { BIPS_BASE } from 'constants/index'
```

**Review by namgold — APPROVED** [review]

File: src/pages/KyberDAO/Vote/ProposalItem/VoteInformation.tsx (hunk 1 above)
Comment (viet-nv) [inline src/pages/KyberDAO/Vote/ProposalItem/VoteInformation.tsx:19]: please use absolute import

Comment (nhd98z) [reply src/pages/KyberDAO/Vote/ProposalItem/VoteInformation.tsx:19]: Fixed. Can't use absolute for constants.

File: src/pages/KyberDAO/Vote/ProposalItem/VoteInformation.tsx [hunk 2]
```diff
@@ -78,7 +91,20 @@ export default function VoteInformation({ proposal }: { proposal: ProposalDetail
...
+          <Text color={theme.text}>{t`Reached`}</Text>
+        ) : (
+          <MouseoverTooltip
+            text={`Total amount required: ${Math.floor(+totalAmountRequired.toFixed(0)).toLocaleString()} KNC`}
+            placement="bottom"
+            width="fit-content"
+          >
+            <Row width="fit-content" gap="6px" color={theme.warning}>
+              <WarningIcon size="16" solid />
+              <Text color={theme.warning} fontWeight={500}>{t`Not Reached`}</Text>
```
Comment (viet-nv) [inline src/pages/KyberDAO/Vote/ProposalItem/VoteInformation.tsx:104]: <Trans>Not Reached</Trans>

File: src/pages/KyberDAO/Vote/ProposalItem/VoteInformation.tsx (hunk 1 above)
Comment (nhd98z) [reply src/pages/KyberDAO/Vote/ProposalItem/VoteInformation.tsx]: Fixed.

File: src/pages/KyberDAO/Vote/ProposalItem/VoteInformation.tsx (hunk 2 above)
Comment (nhd98z) [reply src/pages/KyberDAO/Vote/ProposalItem/VoteInformation.tsx:103]: Yes?

Comment (viet-nv) [reply src/pages/KyberDAO/Vote/ProposalItem/VoteInformation.tsx:103]: `<Trans>Not Reached</Trans>`

Comment (viet-nv) [reply src/pages/KyberDAO/Vote/ProposalItem/VoteInformation.tsx:103]: https://lingui.dev/tutorials/react-patterns

Comment (nhd98z) [reply src/pages/KyberDAO/Vote/ProposalItem/VoteInformation.tsx]: Fixed.

**Review by viet-nv — APPROVED** [review]

**Pull request merged**